CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 - Fix typo for Japanese translation @terapyon
 - Fix refresh of the `Breadcrumbs` and `Navigation` components on calling `/login` and `/logout` @sneridagh
+- Breaking: change component to where page title is set to `App.jsx` @jackahl
```

Comment:
@jackahl this needs to go into the "breaking" section. We also need to add a section to our upgrade guide. Maybe this should go into Volto 6. What do you think @sneridagh?